src/widget/sidebar_filler.h

```diff
@@ -0,0 +1,13 @@
+#ifndef WIDGET_SIDEBAR_FILLER_H
+#define WIDGET_SIDEBAR_FILLER_H
+
+#include "input/mouse.h"
+
+void sidebar_filler_draw_foreground(int x_offset, int y_offset, int sidebar_width, int is_collapsed);
+void sidebar_filler_draw_background(int x_offset, int y_offset, int sidebar_width, int is_collapsed, int sidebar_height);
```

Comment:
Rename `sidebar_width` to just `width`, and `sidebar_height` to `available_height` to make it more clear what the variables should be (see my comment in `sidebar.c`).
Also, place the width and height arguments next to each other, since they're related.